dev-notes/JavaSDK_With_DogfoodSubscriptions.md

```diff
@@ -0,0 +1,40 @@
+
+### Using Java SDK with Dogfood subscriptions
+
+If you have a dogfood subscription for ARM endpoint https://api-dogfood.resources.windows-int.net/ then first you will need to build an AzureEnvironment instance for Canary:
```

Comment:
Can you add a link to instructions on how to get a subscription whitelisted for access to the dogfood endpoint?

Reply:
as discussed this is not added intentionally.